playbooks/roles/wordpress/templates/default-nginx.j2

```diff
@@ -9,8 +9,9 @@ server {
 server {
         listen 443 ssl;
         server_name {{ DEFAULT_SITE_HOSTNAME }};
-        ssl_certificate /etc/letsencrypt/live/{{ DEFAULT_SITE_HOSTNAME }}/fullchain.pem;
-        ssl_certificate_key /etc/letsencrypt/live/{{ DEFAULT_SITE_HOSTNAME }}/privkey.pem;
+        # default site uses landing site certificate
```

Comment:
@alanoe, is this because there is just a single certificate and the landing site domain is the primary domain on the certificate? If yes, can you add a sentence or two to clarify that here and in all the similar comments in the files below?

Reply:
@lgp171188 Yes. I'll add it to the doc.